.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: Build master
```

Comment:
[This](https://github.com/dpanfilyonok/GraphBLAS-sharp/blob/github-actions/.github/workflows/build.yml) windows build script works for me.